packages/core/useStyleTag/index.md

```diff
@@ -4,7 +4,7 @@ category: Browser
 
 # useStyleTag
 
-Inject reactive `&lt;style&gt;` element in head.
+Inject reactive `<style>` element in head.
```

Comment:
@antfu This characters were causing this issue on function search:

![image](https://user-images.githubusercontent.com/19991745/151024788-034618a7-179f-44d2-9b1d-3935d906256d.png)

Reply:
could fix on the function list side. In `<code>` `&lt;` won't come back to `<`